2015-08-14T05:57:04+02:00 EMERG (0):   1 MB | Pimcore\Config::getSystemConfig() [39]: Cannot find system configuration, should be located at: C:\xampp\htdocs\boneeto/website/var/config/system.xml
2015-08-14T05:57:04+02:00 EMERG (0):   2 MB | Pimcore\Config::getSystemConfig() [39]: Cannot find system configuration, should be located at: C:\xampp\htdocs\boneeto/website/var/config/system.xml
2015-08-14T05:57:05+02:00 EMERG (0):   2 MB | Pimcore\Config::getSystemConfig() [39]: Cannot find system configuration, should be located at: C:\xampp\htdocs\boneeto/website/var/config/system.xml
2015-08-14T05:57:05+02:00 EMERG (0):   2 MB | Pimcore\Config::getSystemConfig() [39]: Cannot find system configuration, should be located at: C:\xampp\htdocs\boneeto/website/var/config/system.xml
2015-08-14T05:57:05+02:00 EMERG (0):   1 MB | Pimcore\Config::getSystemConfig() [39]: Cannot find system configuration, should be located at: C:\xampp\htdocs\boneeto/website/var/config/system.xml
2015-08-14T05:57:05+02:00 EMERG (0):   2 MB | Pimcore\Config::getSystemConfig() [39]: Cannot find system configuration, should be located at: C:\xampp\htdocs\boneeto/website/var/config/system.xml
2015-08-14T05:57:05+02:00 EMERG (0):   2 MB | Pimcore\Config::getSystemConfig() [39]: Cannot find system configuration, should be located at: C:\xampp\htdocs\boneeto/website/var/config/system.xml
2015-08-14T05:57:05+02:00 EMERG (0):   2 MB | Pimcore\Config::getSystemConfig() [39]: Cannot find system configuration, should be located at: C:\xampp\htdocs\boneeto/website/var/config/system.xml
2015-08-14T05:58:07+02:00 EMERG (0):   1 MB | Pimcore\Config::getSystemConfig() [39]: Cannot find system configuration, should be located at: C:\xampp\htdocs\boneeto/website/var/config/system.xml
2015-08-14T05:58:07+02:00 EMERG (0):   2 MB | Pimcore\Config::getSystemConfig() [39]: Cannot find system configuration, should be located at: C:\xampp\htdocs\boneeto/website/var/config/system.xml
2015-08-14T05:58:07+02:00 EMERG (0):   2 MB | Pimcore\Config::getSystemConfig() [39]: Cannot find system configuration, should be located at: C:\xampp\htdocs\boneeto/website/var/config/system.xml
2015-08-14T05:58:07+02:00 EMERG (0):   2 MB | Pimcore\Config::getSystemConfig() [39]: Cannot find system configuration, should be located at: C:\xampp\htdocs\boneeto/website/var/config/system.xml
2015-08-14T05:58:07+02:00 DEBUG (7):   4 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 2
2015-08-14T05:59:03+02:00 DEBUG (7):   4 MB | Pimcore\Resource\Wrapper->closeConnectionResource() [119]: Zend_Db_Adapter_Pdo_Mysql: closing MySQL-Server connection with ID: 2
2015-08-14T05:59:03+02:00 DEBUG (7):   4 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 3
2015-08-14T05:59:20+02:00 DEBUG (7):   4 MB | Pimcore\Model\Cache::load() [289]: Key pimcore_system_resource_columns_users doesn't exist in cache
2015-08-14T05:59:21+02:00 ERR (3):   5 MB | Pimcore\Model\Cache::storeToCache() [426]: Failed to add entry pimcore_system_resource_columns_users to the cache, item-size was 370 B
2015-08-14T05:59:21+02:00 DEBUG (7):   5 MB | Pimcore\Model\Cache::storeToCache() [429]: Added pimcore_system_resource_columns_users to cache
2015-08-14T05:59:21+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T05:59:21+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 4
2015-08-14T05:59:22+02:00 ERR (3):   5 MB | Pimcore\Controller\Action\Admin->init() [127]: Prevented access to /admin/ because there is no user in the session!
2015-08-14T05:59:22+02:00 ERR (3):   5 MB | Array
(
    [server] => Array
        (
            [REDIRECT_MIBDIRS] => C:/xampp/php/extras/mibs
            [REDIRECT_MYSQL_HOME] => \xampp\mysql\bin
            [REDIRECT_OPENSSL_CONF] => C:/xampp/apache/bin/openssl.cnf
            [REDIRECT_PHP_PEAR_SYSCONF_DIR] => \xampp\php
            [REDIRECT_PHPRC] => \xampp\php
            [REDIRECT_TMP] => \xampp\tmp
            [REDIRECT_STATUS] => 200
            [MIBDIRS] => C:/xampp/php/extras/mibs
            [MYSQL_HOME] => \xampp\mysql\bin
            [OPENSSL_CONF] => C:/xampp/apache/bin/openssl.cnf
            [PHP_PEAR_SYSCONF_DIR] => \xampp\php
            [PHPRC] => \xampp\php
            [TMP] => C:\xampp\htdocs\boneeto/website/var/cache
            [HTTP_HOST] => boneeto
            [HTTP_USER_AGENT] => Mozilla/5.0 (Windows NT 6.3; WOW64; rv:40.0) Gecko/20100101 Firefox/40.0
            [HTTP_ACCEPT] => text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8
            [HTTP_ACCEPT_LANGUAGE] => en-US,en;q=0.5
            [HTTP_ACCEPT_ENCODING] => gzip, deflate
            [HTTP_REFERER] => http://boneeto/install/
            [HTTP_CONNECTION] => keep-alive
            [PATH] => C:\Program Files (x86)\ActiveState Komodo Edit 9 nightly\;C:\ProgramData\Oracle\Java\javapath;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Program Files (x86)\Intel\iCLS Client\;C:\Program Files\Intel\iCLS Client\;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Program Files\Intel\Intel(R) Management Engine Components\DAL;C:\Program Files\Intel\Intel(R) Management Engine Components\IPT;C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\DAL;C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\IPT;C:\Program Files\SourceGear\Common\DiffMerge\;C:\Program Files (x86)\Rational\common
            [SystemRoot] => C:\xampp\htdocs\boneeto/website/var/cache
            [COMSPEC] => C:\Windows\system32\cmd.exe
            [PATHEXT] => .COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC
            [WINDIR] => C:\Windows
            [SERVER_SIGNATURE] => <address>Apache/2.4.10 (Win32) OpenSSL/1.0.1i PHP/5.5.15 Server at boneeto Port 80</address>

            [SERVER_SOFTWARE] => Apache/2.4.10 (Win32) OpenSSL/1.0.1i PHP/5.5.15
            [SERVER_NAME] => boneeto
            [SERVER_ADDR] => 127.0.0.1
            [SERVER_PORT] => 80
            [REMOTE_ADDR] => 127.0.0.1
            [DOCUMENT_ROOT] => C:/xampp/htdocs/boneeto
            [REQUEST_SCHEME] => http
            [CONTEXT_PREFIX] => 
            [CONTEXT_DOCUMENT_ROOT] => C:/xampp/htdocs/boneeto
            [SERVER_ADMIN] => postmaster@dummy-host.localhost
            [SCRIPT_FILENAME] => C:/xampp/htdocs/boneeto/index.php
            [REMOTE_PORT] => 55098
            [REDIRECT_URL] => /admin/
            [GATEWAY_INTERFACE] => CGI/1.1
            [SERVER_PROTOCOL] => HTTP/1.1
            [REQUEST_METHOD] => GET
            [QUERY_STRING] => 
            [REQUEST_URI] => /admin/
            [SCRIPT_NAME] => /index.php
            [PHP_SELF] => /index.php
            [REQUEST_TIME_FLOAT] => 1439524761.454
            [REQUEST_TIME] => 1439524761
            [TMPDIR] => C:\xampp\htdocs\boneeto/website/var/cache
            [TEMP] => C:\xampp\htdocs\boneeto/website/var/cache
            [windir] => C:\xampp\htdocs\boneeto/website/var/cache
        )

    [get] => Array
        (
        )

    [post] => Array
        (
        )

    [session] => 
    [cookie] => Array
        (
        )

)

2015-08-14T05:59:22+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T05:59:22+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 5
2015-08-14T05:59:35+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T05:59:35+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 6
2015-08-14T05:59:35+02:00 DEBUG (7):   6 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 7
2015-08-14T05:59:35+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T05:59:35+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 8
2015-08-14T05:59:36+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 9
2015-08-14T05:59:36+02:00 INFO (6):   6 MB | Pimcore\Model\Schedule\Manager\Factory::getManager() [47]: Using Schedule\Manager\Procedural as maintenance manager
2015-08-14T05:59:36+02:00 WARN (4):   7 MB | Pimcore\Video\Adapter\Ffmpeg->isAvailable() [50]: [Exception] with message: No ffmpeg executable found, please configure the correct path in the system settings
In file: C:\xampp\htdocs\boneeto\pimcore\lib\Pimcore\Video\Adapter\Ffmpeg.php on line 84
#0 C:\xampp\htdocs\boneeto\pimcore\lib\Pimcore\Video\Adapter\Ffmpeg.php(44): Pimcore\Video\Adapter\Ffmpeg::getFfmpegCli()
#1 C:\xampp\htdocs\boneeto\pimcore\lib\Pimcore\Video.php(69): Pimcore\Video\Adapter\Ffmpeg->isAvailable()
#2 C:\xampp\htdocs\boneeto\pimcore\lib\Pimcore\Video.php(51): Pimcore\Video::getDefaultAdapter()
#3 C:\xampp\htdocs\boneeto\pimcore\modules\admin\views\scripts\index\index.php(509): Pimcore\Video::isAvailable()
#4 C:\xampp\htdocs\boneeto\pimcore\lib\Zend\View.php(108): include('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\boneeto\pimcore\lib\Zend\View\Abstract.php(888): Zend_View->_run('C:/xampp/htdocs...')
#6 C:\xampp\htdocs\boneeto\pimcore\lib\Zend\Controller\Action\Helper\ViewRenderer.php(904): Zend_View_Abstract->render('index/index.php')
#7 C:\xampp\htdocs\boneeto\pimcore\lib\Zend\Controller\Action\Helper\ViewRenderer.php(925): Zend_Controller_Action_Helper_ViewRenderer->renderScript('index/index.php', NULL)
#8 C:\xampp\htdocs\boneeto\pimcore\lib\Zend\Controller\Action\Helper\ViewRenderer.php(964): Zend_Controller_Action_Helper_ViewRenderer->render()
#9 C:\xampp\htdocs\boneeto\pimcore\lib\Pimcore\Controller\Action\Helper\ViewRenderer.php(42): Zend_Controller_Action_Helper_ViewRenderer->postDispatch()
#10 C:\xampp\htdocs\boneeto\pimcore\lib\Zend\Controller\Action\HelperBroker.php(277): Pimcore\Controller\Action\Helper\ViewRenderer->postDispatch()
#11 C:\xampp\htdocs\boneeto\pimcore\lib\Zend\Controller\Action.php(527): Zend_Controller_Action_HelperBroker->notifyPostDispatch()
#12 C:\xampp\htdocs\boneeto\pimcore\lib\Zend\Controller\Dispatcher\Standard.php(308): Zend_Controller_Action->dispatch('indexAction')
#13 C:\xampp\htdocs\boneeto\pimcore\lib\Zend\Controller\Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
#14 C:\xampp\htdocs\boneeto\pimcore\lib\Pimcore.php(285): Zend_Controller_Front->dispatch()
#15 C:\xampp\htdocs\boneeto\index.php(18): Pimcore::run()
#16 {main}
2015-08-14T05:59:41+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T05:59:41+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T05:59:41+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 10
2015-08-14T05:59:41+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 11
2015-08-14T05:59:41+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T05:59:41+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 12
2015-08-14T05:59:41+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T05:59:41+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 13
2015-08-14T05:59:41+02:00 DEBUG (7):   6 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 14
2015-08-14T05:59:42+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 16
2015-08-14T05:59:42+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 15
2015-08-14T05:59:42+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 17
2015-08-14T05:59:42+02:00 DEBUG (7):   6 MB | Pimcore\Model\Cache::load() [289]: Key pimcore_system_supported_locales_en doesn't exist in cache
2015-08-14T05:59:42+02:00 DEBUG (7):   6 MB | Pimcore\Model\Cache::load() [289]: Key pimcore_translations_admin_data doesn't exist in cache
2015-08-14T05:59:42+02:00 DEBUG (7):   6 MB | Pimcore\Model\Cache::storeToCache() [429]: Added pimcore_translations_admin_data to cache
2015-08-14T05:59:43+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T05:59:43+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 18
2015-08-14T05:59:44+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 19
2015-08-14T05:59:44+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T05:59:44+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 20
2015-08-14T05:59:45+02:00 DEBUG (7):   6 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 21
2015-08-14T06:01:02+02:00 DEBUG (7):   6 MB | Pimcore\Model\Cache::storeToCache() [429]: Added pimcore_system_supported_locales_en to cache
2015-08-14T06:01:04+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:01:04+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:01:04+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:01:04+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:01:04+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 23
2015-08-14T06:01:04+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:01:04+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 22
2015-08-14T06:01:04+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 24
2015-08-14T06:01:04+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 25
2015-08-14T06:01:04+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 26
2015-08-14T06:01:04+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:01:04+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 27
2015-08-14T06:01:04+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 28
2015-08-14T06:01:05+02:00 DEBUG (7):   6 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 29
2015-08-14T06:01:05+02:00 DEBUG (7):   6 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 30
2015-08-14T06:01:05+02:00 DEBUG (7):   6 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 31
2015-08-14T06:01:05+02:00 DEBUG (7):   7 MB | Pimcore\Model\Cache::load() [287]: Successfully got data for key pimcore_system_supported_locales_en from cache
2015-08-14T06:01:05+02:00 DEBUG (7):   6 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 32
2015-08-14T06:01:05+02:00 DEBUG (7):   6 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 33
2015-08-14T06:01:05+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:01:05+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 34
2015-08-14T06:01:05+02:00 DEBUG (7):   7 MB | Pimcore\Model\Cache::load() [289]: Key pimcore_document_1 doesn't exist in cache
2015-08-14T06:01:05+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:01:05+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 35
2015-08-14T06:01:05+02:00 DEBUG (7):   6 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 36
2015-08-14T06:01:05+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:01:05+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 37
2015-08-14T06:01:05+02:00 DEBUG (7):   7 MB | Pimcore\Model\Cache::load() [289]: Key pimcore_asset_1 doesn't exist in cache
2015-08-14T06:01:06+02:00 DEBUG (7):   6 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 38
2015-08-14T06:01:06+02:00 DEBUG (7):   7 MB | Pimcore\Model\Cache::load() [289]: Key pimcore_object_1 doesn't exist in cache
2015-08-14T06:01:06+02:00 DEBUG (7):   7 MB | Pimcore\Model\Cache::storeToCache() [393]: prepared Pimcore\Model\Document\Page 1 for data cache with tags: document_1
2015-08-14T06:01:06+02:00 DEBUG (7):   7 MB | Pimcore\Model\Cache::storeToCache() [429]: Added pimcore_document_1 to cache
2015-08-14T06:01:06+02:00 DEBUG (7):   7 MB | Pimcore\Model\Cache::storeToCache() [393]: prepared Pimcore\Model\Object\Folder 1 for data cache with tags: object_1
2015-08-14T06:01:06+02:00 DEBUG (7):   7 MB | Pimcore\Model\Cache::storeToCache() [429]: Added pimcore_object_1 to cache
2015-08-14T06:01:06+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:01:06+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 39
2015-08-14T06:01:06+02:00 DEBUG (7):   7 MB | Pimcore\Model\Cache::storeToCache() [393]: prepared Pimcore\Model\Asset\Folder 1 for data cache with tags: asset_1
2015-08-14T06:01:06+02:00 DEBUG (7):   7 MB | Pimcore\Model\Cache::storeToCache() [429]: Added pimcore_asset_1 to cache
2015-08-14T06:01:06+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:01:07+02:00 DEBUG (7):   6 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 41
2015-08-14T06:01:07+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 40
2015-08-14T06:01:07+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:01:07+02:00 DEBUG (7):   7 MB | Pimcore\Model\Cache::load() [287]: Successfully got data for key pimcore_document_1 from cache
2015-08-14T06:01:07+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 42
2015-08-14T06:01:07+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:01:07+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 43
2015-08-14T06:01:07+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 44
2015-08-14T06:01:07+02:00 DEBUG (7):   6 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 45
2015-08-14T06:01:07+02:00 DEBUG (7):   6 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 46
2015-08-14T06:01:07+02:00 DEBUG (7):   7 MB | Pimcore\Model\Cache::load() [287]: Successfully got data for key pimcore_object_1 from cache
2015-08-14T06:01:07+02:00 DEBUG (7):   7 MB | Pimcore\Model\Cache::load() [287]: Successfully got data for key pimcore_asset_1 from cache
2015-08-14T06:01:07+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 47
2015-08-14T06:02:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:02:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 48
2015-08-14T06:02:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 49
2015-08-14T06:03:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:03:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 50
2015-08-14T06:03:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 51
2015-08-14T06:04:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:04:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 52
2015-08-14T06:04:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 53
2015-08-14T06:05:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:05:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 54
2015-08-14T06:05:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 55
2015-08-14T06:06:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:06:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 56
2015-08-14T06:06:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 57
2015-08-14T06:07:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:07:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 60
2015-08-14T06:07:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 61
2015-08-14T06:08:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:08:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 62
2015-08-14T06:08:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 63
2015-08-14T06:09:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:09:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 64
2015-08-14T06:09:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 65
2015-08-14T06:10:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:10:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 66
2015-08-14T06:10:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 67
2015-08-14T06:11:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:11:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 68
2015-08-14T06:11:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 69
2015-08-14T06:12:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:12:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 70
2015-08-14T06:12:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 71
2015-08-14T06:13:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:13:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 72
2015-08-14T06:13:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 73
2015-08-14T06:14:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:14:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 74
2015-08-14T06:14:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 75
2015-08-14T06:15:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:15:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 76
2015-08-14T06:15:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 77
2015-08-14T06:16:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:16:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 78
2015-08-14T06:16:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 79
2015-08-14T06:17:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:17:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 80
2015-08-14T06:17:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 81
2015-08-14T06:18:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:18:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 82
2015-08-14T06:18:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 83
2015-08-14T06:19:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:19:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 84
2015-08-14T06:19:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 85
2015-08-14T06:20:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:20:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 86
2015-08-14T06:20:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 87
2015-08-14T06:21:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:21:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 88
2015-08-14T06:21:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 89
2015-08-14T06:22:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:22:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 92
2015-08-14T06:22:03+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 93
2015-08-14T06:23:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:23:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 98
2015-08-14T06:23:03+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 99
2015-08-14T06:24:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:24:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 100
2015-08-14T06:24:03+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 101
2015-08-14T06:25:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:25:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 102
2015-08-14T06:25:03+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 103
2015-08-14T06:26:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:26:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 104
2015-08-14T06:26:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 105
2015-08-14T06:27:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:27:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 106
2015-08-14T06:27:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 107
2015-08-14T06:28:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:28:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 108
2015-08-14T06:28:03+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 109
2015-08-14T06:29:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:29:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 110
2015-08-14T06:29:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 111
2015-08-14T06:30:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:30:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 112
2015-08-14T06:30:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 113
2015-08-14T06:31:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:31:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 114
2015-08-14T06:31:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 115
2015-08-14T06:32:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:32:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 116
2015-08-14T06:32:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 117
2015-08-14T06:33:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:33:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 118
2015-08-14T06:33:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 119
2015-08-14T06:34:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:34:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 120
2015-08-14T06:34:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 121
2015-08-14T06:35:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:35:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 122
2015-08-14T06:35:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 123
2015-08-14T06:36:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:36:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 124
2015-08-14T06:36:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 125
2015-08-14T06:37:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:37:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 126
2015-08-14T06:37:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 127
2015-08-14T06:38:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:38:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 128
2015-08-14T06:38:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 129
2015-08-14T06:39:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:39:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 130
2015-08-14T06:39:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 131
2015-08-14T06:40:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:40:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 132
2015-08-14T06:40:03+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 133
2015-08-14T06:41:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:41:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 134
2015-08-14T06:41:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 135
2015-08-14T06:42:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:42:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 136
2015-08-14T06:42:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 137
2015-08-14T06:43:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:43:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 138
2015-08-14T06:43:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 139
2015-08-14T06:44:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:44:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 140
2015-08-14T06:44:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 141
2015-08-14T06:45:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:45:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 142
2015-08-14T06:45:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 143
2015-08-14T06:46:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:46:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 144
2015-08-14T06:46:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 145
2015-08-14T06:47:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:47:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 146
2015-08-14T06:47:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 147
2015-08-14T06:48:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:48:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 148
2015-08-14T06:48:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 149
2015-08-14T06:49:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:49:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 150
2015-08-14T06:49:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 151
2015-08-14T06:50:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:50:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 152
2015-08-14T06:50:03+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 153
2015-08-14T06:51:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:51:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 154
2015-08-14T06:51:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 155
2015-08-14T06:52:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:52:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 156
2015-08-14T06:52:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 157
2015-08-14T06:53:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:53:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 158
2015-08-14T06:53:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 159
2015-08-14T06:54:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:54:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 160
2015-08-14T06:54:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 161
2015-08-14T06:55:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:55:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 162
2015-08-14T06:55:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 163
2015-08-14T06:56:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:56:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 164
2015-08-14T06:56:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 165
2015-08-14T06:57:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:57:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 166
2015-08-14T06:57:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 167
2015-08-14T06:58:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:58:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 168
2015-08-14T06:58:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 169
2015-08-14T06:59:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T06:59:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 170
2015-08-14T06:59:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 171
2015-08-14T07:00:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T07:00:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 172
2015-08-14T07:00:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 173
2015-08-14T07:01:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T07:01:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 174
2015-08-14T07:01:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 175
2015-08-14T07:02:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T07:02:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 176
2015-08-14T07:02:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 177
2015-08-14T07:03:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T07:03:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 178
2015-08-14T07:03:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 179
2015-08-14T07:04:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T07:04:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 180
2015-08-14T07:04:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 181
2015-08-14T07:05:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T07:05:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 182
2015-08-14T07:05:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 183
2015-08-14T07:06:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T07:06:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 184
2015-08-14T07:06:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 185
2015-08-14T07:07:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T07:07:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 186
2015-08-14T07:07:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 187
2015-08-14T07:08:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T07:08:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 188
2015-08-14T07:08:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 189
2015-08-14T07:09:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T07:09:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 190
2015-08-14T07:09:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 191
2015-08-14T07:10:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T07:10:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 192
2015-08-14T07:10:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 193
2015-08-14T07:11:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T07:11:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 194
2015-08-14T07:11:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 195
2015-08-14T07:12:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T07:12:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 196
2015-08-14T07:12:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 197
2015-08-14T07:13:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T07:13:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 198
2015-08-14T07:13:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 199
2015-08-14T07:14:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T07:14:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 200
2015-08-14T07:14:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 201
2015-08-14T07:15:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T07:15:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 202
2015-08-14T07:15:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 203
2015-08-14T07:16:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T07:16:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 204
2015-08-14T07:16:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 205
2015-08-14T07:17:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T07:17:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 206
2015-08-14T07:17:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 207
2015-08-14T07:18:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T07:18:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 208
2015-08-14T07:18:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 209
2015-08-14T07:19:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T07:19:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 210
2015-08-14T07:19:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 211
2015-08-14T07:20:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T07:20:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 212
2015-08-14T07:20:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 213
2015-08-14T07:21:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T07:21:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 214
2015-08-14T07:21:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 215
2015-08-14T07:22:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T07:22:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 216
2015-08-14T07:22:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 217
2015-08-14T07:23:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T07:23:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 218
2015-08-14T07:23:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 219
2015-08-14T07:24:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T07:24:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 220
2015-08-14T07:24:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 221
2015-08-14T07:25:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T07:25:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 222
2015-08-14T07:25:03+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 223
2015-08-14T07:26:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T07:26:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 224
2015-08-14T07:26:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 225
2015-08-14T07:27:02+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T07:27:02+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 226
2015-08-14T07:27:02+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 227
2015-08-14T07:42:27+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T07:42:27+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T07:42:27+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 228
2015-08-14T07:42:27+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 229
2015-08-14T07:42:27+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 230
2015-08-14T07:42:27+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 231
2015-08-14T07:43:27+02:00 DEBUG (7):   2 MB | Pimcore\Cache\Backend\MysqlTable->getDb() [46]: Initialize dedicated MySQL connection for the cache adapter
2015-08-14T07:43:27+02:00 DEBUG (7):   2 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 232
2015-08-14T07:43:27+02:00 DEBUG (7):   5 MB | Pimcore\Resource\Mysql::getConnection() [95]: Zend_Db_Adapter_Pdo_Mysql: Successfully established connection to MySQL-Server, Process-ID: 233
